apiserver/clusterservice/scaleimpl.go

```diff
@@ -240,6 +275,11 @@ func ScaleQuery(name, ns string) msgs.ScaleQueryResponse {
 		return response
 	}
 
+	// indicate in the response whether or not a standby cluster
+	if cluster.Spec.Standby {
+		response.Standby = true
+	}
```

Comment:
`response.Standby = cluster.Spec.Standby` :smile:

Reply:
Updated to `response.Standby = cluster.Spec.Standby`.